src/ngraph/op/convolution.cpp

```diff
@@ -45,6 +45,16 @@ op::v1::Convolution::Convolution(const Output<Node>& data_batch,
     constructor_validate_and_infer_types();
 }
 
+bool op::v1::Convolution::visit_attributes(AttributeVisitor& visitor)
+{
+    visitor.on_attribute("strides", m_strides);
+    visitor.on_attribute("dilations", m_dilations);
+    visitor.on_attribute("pads_begin", m_pads_begin);
+    visitor.on_attribute("pads_end", m_pads_end);
+    visitor.on_attribute("auto_pad", m_auto_pad);
```

Comment:
What is the default value of auto_pad in this case?

We are using ngraph::op::PadType::EXPLICIT as default value. Could we specify it here?

Reply:
Can't specify it here, and we can't default it because it's framework specific. We could have something to set a global to be used as a default.